.github/workflows/trigger_prerelease.yml

```diff
@@ -0,0 +1,46 @@
+name: Trigger prerelease creation
+
+# This workflow triggers a prerelease creation with changelog and the release notes created by the release toolkit.
+# This workflow should be triggered merely from the default branch.
+# If you wish to be 100% free creating a prerelease, just create it manually.
+
+on:
+  workflow_dispatch:
+## TODO here we will need a cronjob.
+
+jobs:
+  prerelease:
+    name: Create prerelease
+    runs-on: ubuntu-latest
+    outputs:
+      version: ${{ steps.version.outputs.next-version }}
+      image-tag: ${{ steps.image-tag.outputs.tag }}
```

Comment:
There are no jobs left in this pipeline that need these outputs.

Reply:
you are right, thanks!